adoc/admin-software-installation.adoc

```diff
@@ -188,7 +202,76 @@ helm upgrade --tls <RELEASE_NAME> <CHART> [flags]
 helm del --tls <RELEASE_NAME> [flags]
 ----
 
+[[helm-2to3-migration]]
+=== Helm 2 to 3 Migration
+[NOTE]S
+====
+The process for migrating an installation from Helm 2 to Helm 3 has been documented and tested by the Helm community. Please reference the following links before proceeding.
+
+- https://v3.helm.sh/docs/topics/v2_v3_migration/
+- https://helm.sh/blog/migrate-from-helm-v2-to-helm-v3/
+- https://github.com/helm/helm-2to3
+
+====
+
+==== Preconditions
+
+* A healthy CaaSP 5.0 installation with applications deployed using Helm 2 and Tiller.
+* A system, which `skuba` and `helm` version 2 have run on previously.
+** The procedure below requires an available internet connection to install the `2to3` plugin.  If the installation is in an air gapped environment, the system may need to be moved back out of the air gapped environment.
+* These instructions are written for a single cluster managed from a single Helm 2 installation. If more than one cluster is being managed by this installation of Helm 2, please reference https://github.com/helm/helm-2to3 for further details and do not do the clean-up step until all clusters are migrated.
+
+
+==== Migration Procedure
+
+This is a procedure for migrating a CaaSP 5.0 deployment that has used Helm 2 to deploy applications.
+
+. Install `helm3` package in the same location you normally run `skuba` commands (alongside the helm2 package):
++
+----
+sudo zypper in helm3
+----
+. Install the `2to3` plugin:
++
+----
+helm3 plugin install https://github.com/helm/helm-2to3.git
+----
+. Backup Helm 2 data found in the following:
+.. Helm 2 home folder.
+.. Release data from the cluster. Refer to link:http://technosophos.com/2017/03/23/how-helm-uses-configmaps-to-store-data.html[How Helm Uses ConfigMaps to Store Data] for details on how Helm 2 stores release data in the cluster. This should apply similarly if Helm 2 is configured for secrets.
```

Comment:
@jodavis-suse I think this is the article you wanted to link, right? Your text did not contain the link, but I found an article of that exact name, just checking that I'm not linking something unrelated.

Reply:
Yes, thanks, that got lost in my cut and paste over from confluence.  I fixed it there but didn't get it to here. :)